packages/core/test/template-engine/built-in-extensions/cache-layer/cache.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { createTestCompiler } from '../../testCompiler';
+
+it('If exist arguments behind "cache" symbol, extension should throw an error', async () => {
+  // Arrange
+  const { compiler } = await createTestCompiler();
+
+  // Action, Assert
+  await expect(
+    compiler.compile(`{% cache variable %} some statement {% endcache %}`)
+  ).rejects.toThrow(`Expected a block end, but got symbol`);
```

Comment:
just curious, what will happen if there exist arguments behind the "endcache" symbol

Reply:
Thanks for asking the question, it depends on the Nunjucks parser rule, and according to the test case, I add after you asked, looks like it will throw an error caused by Nunjucks.

For making our other members also know the case, I will add the test case!